observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -5,3 +5,143 @@ include::_attributes/common-attributes.adoc[]
+
+== Configuring Outputs
+
+Outputs are configured in an array under `spec.outputs`. Each output must have a unique name and a type. Supported types include:
+
+elasticsearch:: Forwards logs to an external Elasticsearch instance.
+fluentdForward:: Forwards logs to an external Fluentd instance.
+syslog:: Forwards logs to an external syslog server.
+kafka:: Forwards logs to a Kafka broker.
+loki:: Forwards logs to a Loki logging backend.
+cloudwatch:: Forwards logs to AWS Cloudwatch.
+
+Each output type has its own configuration fields. Common ones include:
+
+url:: URL of the output destination.
+secret:: Name of an OpenShift secret containing authentication credentials for the output.
+tls:: TLS client configuration for the output.
+
+The `outputDefaults` field sets global options for outputs like buffer parameters.
+
+== Configuring Pipelines
+
+Pipelines are configured in an array under `spec.pipelines`. Each pipeline must have a unique name and consists of:
+
+inputRefs:: Names of inputs whose logs should be forwarded to this pipeline.
+outputRefs:: Names of outputs to send logs to.
+filterRefs:: (optional) Names of filters to apply.
+
+The order of filterRefs matters, as they are applied sequentially. Earlier filters can drop messages that will not be processed by later filters.
+
+== Configuring Filters
+
+Filters are configured in an array under `spec.filters`. They can match incoming log messages based on the value of structured fields and modify or drop them.
+
+Users can configure the following types of filters:
+
+labelFilter:: Adds labels to logs matching the filter. Useful for labeling certain types of logs.
+anonymizeFilter:: Masks user-sensitive fields using hash functions. Useful for GDPR compliance.
+nullFilter:: Drops logs matching the filter.
+
+Each filter type has a `match` configuration for matching fields and filter-specific parameters.
+
+== Enabling Output Security
+
+Outputs can be configured to use TLS/SSL by specifying a `secret` containing the TLS certificate and key. The secret can be created using:
+
+[source,terminal]
+----
+$ oc create secret tls my-es-secret --cert=my-es.crt --key=my-es.key
+----
+
+Then reference it in the output configuration:
+
+[source,yaml]
+----
+spec:
+  outputs:
+    - name: my-es
+      type: elasticsearch
+      url: https://elasticsearch.example.com
+      secret:
+        name: my-es-secret
+----
+
+For authentication, outputs can specify a `secret` containing username/password or token credentials:
+
+[source,terminal]
+----
+$ oc create secret generic my-es-secret
+  --from-literal=username=my-username
+  --from-literal=password=my-password
+----
+
+Then reference it in the output configuration:
+
+[source,yaml]
+----
+spec:
+  outputs:
+    - name: my-es
+      type: elasticsearch
+      url: https://elasticsearch.example.com
+      secret:
+        name: my-es-secret
+----
+
+Additional output-specific authentication methods are also supported, like the shared_key for Fluentd.
```

Comment:
This is not correct. Vector doesn't support to send to Fluentd using the [forward](https://docs.fluentd.org/input/forward) protocol. Then, for log forwarding to other fluentd could be used the HTTP input plugin that doesn't support shared_key.